src/components/autocomplete/js/autocompleteController.js

```diff
@@ -641,9 +641,15 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.nextTick(function () {
```

Comment:
@ThomasBurleson Probably a good point, but this is a complaint on the original code, not this PR.